src/metrics.py

```diff
@@ -37,4 +37,4 @@ def update(self, tokens: list[str]):
         self.tokens_used += len(tokens)
 
     def compute(self):
-        return self.tokens_used.float()
+        return self.tokens_used.float()
```

Comment:
Please try to change the CRLF to LF
https://stackoverflow.com/questions/49228693/how-to-change-the-eol-for-all-files-from-crlf-to-lf